packages/core/src/plugins/meta/discogs.ts

```diff
@@ -210,6 +201,51 @@ class DiscogsMetaProvider extends MetaProvider {
     };
   }
 
+  async getSimilarArtists(artist: LastFmArtistInfo | undefined): Promise<SimilarArtist[]> {
+    if (!artist?.similar?.artist) {
+      return Promise.resolve([]);
+    }
+    const similarArtists = artist.similar.artist;
+    return getToken()
```

Comment:
Instead of using `.then` and `.catch`, you can just use `await` and try-catch blocks here and in the functions below.

Reply:
I know. I had both versions locally and found the chaining of `then`/`catch` to be more consistent with rest of the code. I don't mind changing it if you like the alternative better.